os_stub/cryptlib_mbedtls/der/der.c

```diff
@@ -0,0 +1,189 @@
+/**
+ *  Copyright Notice:
+ *  Copyright 2021-2022 DMTF. All rights reserved.
+ *  License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/libspdm/blob/main/LICENSE.md
+ **/
+
+/** @file
+ * DER (Distinguished Encoding Rules) format Handler Wrapper Implementation.
```

Comment:
Is that possible to add some crypto unit test for the new APIs in https://github.com/DMTF/libspdm/tree/main/unit_test/test_crypt?

Reply:
Sure. I will add TCs accordingly.